.travis.yml

```diff
@@ -5,13 +5,13 @@ language: generic
 
 matrix:
   include:
-    - env: PYTHON=python2 PYTHON_VER=  PYTAVE=no SYMPY_VER=1.0 OCT=ppa   DOCTEST=yes COLUMNS=80
-    - env: PYTHON=python3 PYTHON_VER=3 PYTAVE=no SYMPY_VER=1.0 OCT=ppa   DOCTEST=yes COLUMNS=80
-    - env: PYTHON=python2 PYTHON_VER=  PYTAVE=no SYMPY_VER=1.0 OCT=daily DOCTEST=no COLUMNS=80
-    - env: PYTHON=python3 PYTHON_VER=3 PYTAVE=no SYMPY_VER=1.0 OCT=daily DOCTEST=no COLUMNS=80
+    - env: PYTHON=python2 PYTHON_VERSION=  PYTAVE=no  SYMPY_VER=1.0 OCT=ppa   DOCTEST=yes COLUMNS=80
+    - env: PYTHON=python3 PYTHON_VERSION=3 PYTAVE=no  SYMPY_VER=1.0 OCT=ppa   DOCTEST=yes COLUMNS=80
+    - env: PYTHON=python2 PYTHON_VERSION=  PYTAVE=yes SYMPY_VER=1.0 OCT=daily DOCTEST=no  COLUMNS=80
+    - env: PYTHON=python3 PYTHON_VERSION=3 PYTAVE=yes SYMPY_VER=1.0 OCT=daily DOCTEST=no  COLUMNS=80
   allow_failures:
-    - env: PYTHON=python2 PYTHON_VER=  PYTAVE=no SYMPY_VER=1.0 OCT=daily DOCTEST=no COLUMNS=80
-    - env: PYTHON=python3 PYTHON_VER=3 PYTAVE=no SYMPY_VER=1.0 OCT=daily DOCTEST=no COLUMNS=80
+    - env: PYTHON=python2 PYTHON_VERSION=  PYTAVE=yes SYMPY_VER=1.0 OCT=daily DOCTEST=no  COLUMNS=80
+    - env: PYTHON=python3 PYTHON_VERSION=3 PYTAVE=yes SYMPY_VER=1.0 OCT=daily DOCTEST=no  COLUMNS=80
```

Comment:
Yes it is for pytave configure. Well we already used a PYTHON_VER so I thought it's better not to include new variables. Moreover defining a python version for the whole environment should signify that it will be used for everything, doesn't it?